ansible/indy_node/roles/indy_node/tasks/configure.yml

```diff
@@ -141,6 +141,22 @@
     creates: "{{ data_volume_mount }}{{ network_name }}/keys"
   register: init_indy_node_result
 
+- name: Write Node Secrets to File (on Ansible Playbook runner)
+  delegate_to: 127.0.0.1
+  shell: echo "{{ init_indy_node_result.stdout_lines }}" >> "/tmp/indy_node_init_results.txt" 2>&1
+  when: write_node_secrets is defined
+
+- name: Cat Node Secrets File
```

Comment:
Depending on your organisation security requirements, those two lines can be construed from mildly dangerous to a real security problem, because they include (twice) sensitive information in the ansible execution log.
You might want to protect those with another variable, like `log_sensitive_information`, that default to `false`